src/sourabh/nestedSwitchCase/BankAccounts.java

```diff
@@ -0,0 +1,167 @@
+/*
+Assignment - 15 : 9th April'2021
+
+A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+          a) Person can debit or credit amount from respective bank.
+              Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+          b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+          c) A Method to show total credit, debit operations 
+                 output : Total credit operation - 7 times
+                              Total debit operation - 5 times
+
+         d) A Method to show individual bank credit operations.
+                output : ICICI bank Credit Operation - 2
+                         HDFC bank Credit Operation - 2
+		                 SBI bank Credit Operation - 1
+                         BOB bank Credit Operation - 4
+
+         e) A Method to show individual bank debit operations.
+                output : ICICI bank Debit Operation - 2
+                         HDFC bank Debit Operation - 2
+		                 SBI 
+ */
+
+package sourabh.nestedSwitchCase;
+
+import java.util.Scanner;
+
+public class BankAccounts {
+	int iciciBalance=1000;
+	int hdfcBalance=2000;
+	int bobBalance=1000;
+	int sbiBalance=2000;
+	int debitCounticici=0, debitCounthdfc=0, debitCountbob=0, debitCountsbi=0;
+	int creditCounticici=0, creditCounthdfc=0, creditCountbob=0, creditCountsbi=0;
+	void bankOperation(String operation, String bankName, int reqAmt, int depositAmt) {
+		switch(operation) {
+			case "debit":
+				switch (bankName) {
+					case "icici":
+						if(reqAmt<iciciBalance) {
```

Comment:
instead of writing multi-time same code block you can write debit() method which accept the two arguments and you can simply call that method